script/server_bootstrap.sh

```diff
@@ -36,7 +37,7 @@ cd $HOME
 git clone https://github.com/yihangho/SimpleCMS.git --bare
 cd SimpleCMS.git
 mkdir -p $DEPLOYMENT_DIR
-GIT_WORK_TREE=$DEPLOYMENT_DIR git checkout -f master
+GIT_WORK_TREE=$DEPLOYMENT_DIR git checkout -f mcc2015
```

Comment:
This is OK, but FYI, you can actually leave it as master - we just need to push to the remote master branch to deploy: `git push origin mcc2015:master`.